addons/hr/models/hr_employee_public.py

```diff
@@ -44,12 +44,9 @@ class HrEmployeePublic(models.Model):
         ('absent', 'Absent'),
         ('archive', 'Archived'),
         ('out_of_working_hour', 'Out of Working hours')], compute='_compute_presence_state', default='out_of_working_hour')
-    hr_icon_display = fields.Selection([
-        ('presence_present', 'Present'),
-        ('presence_out_of_working_hour', 'Out of Working hours'),
-        ('presence_absent', 'Absent'),
-        ('presence_archive', 'Archived'),
-        ('presence_undetermined', 'Undetermined')], compute='_compute_presence_icon')
+    hr_icon_display = fields.Selection(
+        selection=lambda self: self.env['hr.employee']._fields['hr_icon_display']._description_selection(self.env),
```

Comment:
I'm wondering how it would not break translations, did you test if it was ok ? :)